.github/workflows/update-snapshot-packages.yml

```diff
@@ -15,10 +15,7 @@ jobs:
           node-version: '14.x'
       - name: Update snapshot.js
         run: |
-          yarn upgrade @snapshot-labs/snapshot.js  --latest
-      - name: Update snapshot lock
-        run: |
-          yarn upgrade @snapshot-labs/lock@github:snapshot-labs/lock#master  --latest
```

Comment:
@ChaituVR I think we should publish a new version for lock.js and remove only `@github:snapshot-labs/lock#master`.

Reply:
@samuveth Can we merge this for now? It is blocking for snapshot.js to add new networks, will create a new PR with lock.js from the npm package